terraform/eks/variables.tf

```diff
@@ -20,3 +20,7 @@ variable "eks_cluster_name" {
 variable "mock_endpoint" {
   default = "localhost/put-data"
 }
+
+variable "aoc_base" {
+  default = "oltp"
```

Comment:
what's the other possible values for aoc_base? trying to understand it's meaning.

Reply:
Added comments to the variable. The deployment spec of AOC will be different under different scenarios, this is used to identify the testing purpose.